src/Middleware/StaticFiles/test/UnitTests/DirectoryBrowserMiddlewareTests.cs

```diff
@@ -281,7 +281,7 @@ private async Task HeadDirectory_HeadersButNotBodyServed(string baseUrl, string
 
                 Assert.Equal(HttpStatusCode.OK, response.StatusCode);
                 Assert.Equal("text/html; charset=utf-8", response.Content.Headers.ContentType.ToString());
-                Assert.True(response.Content.Headers.ContentLength == 0);
+                Assert.Null(response.Content.Headers.ContentLength);
```

Comment:
@Tratcher Can you verify this is an ok change? https://github.com/dotnet/corefx/pull/38129 changed the behavior

Reply:
This is fine.